iphone/Classes/KrollContext.m

```diff
@@ -1189,6 +1294,11 @@ - (void)main
   [self bindCallback:@"L" callback:&LCallback];
   [self bindCallback:@"alert" callback:&AlertCallback];
 
+  // create Worker global, instantiated with "new Worker('path/to/file.js')
+  TiStringRef node = TiStringCreateWithUTF8CString("Worker");
+  TiObjectSetProperty(context, globalRef, node, TiObjectMakeConstructor(context, TiWorker_class(context), TiWorker_construct), kTiPropertyAttributeNone, NULL);
```

Comment:
I'm not entirely sure this is correct. First, I think `TiObjectMakeConstructor` has no mapping/analog in raw JSC headers, so this is likely what's causing the iOS compile to fail.

Basically, in Windows we hung the type as an object in the global namespace, so that'd be more like:
```cpp
TiObjectSetProperty(context, globalRef, node, TiObjectMake(context, TiWorker_class(context), NULL), kTiPropertyAttributeNone, NULL);
```

Basically what that's saying is make a JS object that wraps the class definition we have defined in `TiWorker_class`, and hang it as `"Worker"` in the global namespace. (Under the hood `TiWorker_initialize` should get called when the object is created as this type).

Then when you do "new Worker()" it should invoke whatever the class definition pointed at for it's `callAsConstructor` property, which should be set to `TiWorker_construct` in this example - as per my comment above.